workspaces/redhat-argocd/metadata/backstage-community-plugin-redhat-argocd-backend.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: extensions.backstage.io/v1alpha1
+kind: Package
+metadata:
+  name: backstage-community-plugin-redhat-argocd-backend
+  namespace: rhdh
+  title: "Red Hat ArgoCD Backend"
+  links:
+    - url: https://red.ht/rhdh
+      title: Homepage
+    - url: https://issues.redhat.com/browse/RHIDP
+      title: Bugs
+    - title: Source Code
+      url: https://github.com/redhat-developer/rhdh/tree/main/dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd-backend-dynamic
+  annotations:
+    backstage.io/source-location: url
+      https://github.com/redhat-developer/rhdh/tree/main/dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd-backend-dynamic
+  tags: []
+spec:
+  packageName: "@backstage-community/plugin-redhat-argocd-backend"
+  dynamicArtifact: ./dynamic-plugins/dist/backstage-community-plugin-redhat-argocd-backend-dynamic
```

Comment:
please point to the latest builds at https://github.com/redhat-developer/rhdh-plugin-export-overlays/pkgs/container/rhdh-plugin-export-overlays%2Fbackstage-community-plugin-redhat-argocd-backend or https://github.com/redhat-developer/rhdh-plugin-export-overlays/pkgs/container/rhdh-plugin-export-overlays%2Fbackstage-community-plugin-redhat-argocd (same comment for front and backend plugins)